functions/bouncer/example.js

```diff
@@ -0,0 +1,51 @@
+export default {
+  id: 'evt_1Hw96LBvVBgmhQW4NgFZu0hH',
+  object: 'event',
+  api_version: '2020-08-27',
+  created: 1607445105,
+  data: {
+    object: {
+      id: 'pi_1Hw96KBvVBgmhQW4Bs4z54fQ',
+      object: 'payment_intent',
+      amount: 2000,
+      amount_capturable: 0,
+      amount_received: 2000,
+      application: null,
+      application_fee_amount: null,
+      canceled_at: null,
+      cancellation_reason: null,
+      capture_method: 'automatic',
+      charges: [Object],
+      client_secret:
+        'pi_1Hw96KBvVBgmhQW4Bs4z54fQ_secret_MWsIxxq3Io13q56KcF7Rfkyg9',
```

Comment:
do we care about this secret?

Reply:
No, it is no longer valid. Though I will probably delete this file all together

Reply:
Yes. This example.js file will be deleted

Reply:
File removed with commit 87bfb5e